x-pack/plugins/fleet/public/applications/fleet/sections/epm/screens/detail/overview/overview.tsx

```diff
@@ -29,7 +29,7 @@ export const OverviewPage: React.FC<Props> = memo(({ packageInfo }: Props) => {
   return (
     <EuiFlexGroup alignItems="flexStart">
       <LeftColumn grow={2} />
-      <EuiFlexItem grow={9}>
+      <EuiFlexItem grow={9} style={{ wordBreak: 'break-word' }}>
```

Comment:
👍🏻 on the solution, EUI has a set of utility classes that we can use instead of inline styles: https://elastic.github.io/eui/#/utilities/css-utility-classes#text